pandas/core/methods/describe.py

```diff
@@ -364,9 +363,9 @@ def select_describe_func(
         return describe_categorical_1d
 
 
-def refine_percentiles(
+def _refine_percentiles(
```

Comment:
Any reason to privatize this? Everything inside pandas.core is already "private".

Reply:
i like being able to tell when something is only needed inside the module